client/my-sites/stats/activity-log-item/activity-icon.jsx

```diff
@@ -1,55 +1,25 @@
+/** @format */
 /**
  * External dependencies
  */
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import Gridicon from 'gridicons';
-import { head, split } from 'lodash';
 
 export default class ActivityIcon extends PureComponent {
-
 	static propTypes = {
-		activityName: PropTypes.string.isRequired,
 		activityIcon: PropTypes.string.isRequired,
+		activityName: PropTypes.string.isRequired,
```

Comment:
This prop doesn't seem to be used anymore.

Reply:
Thanks for spotting that!